AzureCalling/Views/ParticipantView.swift

```diff
@@ -12,6 +12,9 @@ class ParticipantView: UIView {
 
     // MARK: Properties
 
+    var switchCamera: () -> Void = {}
```

Comment:
Instead of a property, it might be better to have a method to set the callback:
`setOnSwitchCamera()`

Reply:
Meaning we prefer private fields to limit access and have a setter method instead? Got it.